src/test/java/com/pnu/dev/shpaltaif/integration/service/FeedbackServiceIntegrationTest.java

```diff
@@ -0,0 +1,114 @@
+package com.pnu.dev.shpaltaif.integration.service;
+
+import com.pnu.dev.shpaltaif.domain.Feedback;
+import com.pnu.dev.shpaltaif.integration.BaseIntegrationTest;
+import com.pnu.dev.shpaltaif.repository.FeedbackRepository;
+import com.pnu.dev.shpaltaif.service.FeedbackService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+import java.util.stream.IntStream;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class FeedbackServiceIntegrationTest extends BaseIntegrationTest {
+
+    private final static Pageable PAGEABLE = PageRequest.of(0, 10,
+            Sort.by(Sort.Direction.ASC, "reviewed").and(Sort.by(Sort.Direction.DESC, "createdAt")));
+
+    private final static String FEEDBACK_CONTENT = "content";
+
+    private final static String FEEDBACK_USER_INFO = "userInfo";
+
+
+    @Autowired
+    private FeedbackService feedbackService;
+
+    @Autowired
+    private FeedbackRepository feedbackRepository;
+
+    @Test
+    public void createThanReviewThanDeactivateTest() {
+
+        assertEquals(0, feedbackService.countUnreviewed());
+        assertEquals(0, feedbackService.findAll(PAGEABLE).getTotalElements());
+
+        //Create feedback
+        Feedback feedbackToCreate = Feedback.builder()
```

Comment:
We agreed to perform each integration test in transaction, but set up data in the first one, that wouldn't rollback immediately.

Just to follow A-A-A pattern.

But let it be like this for now. Maybe, one day, this test will be rewritten.